### PR TITLE
feat: Footer implementation

### LINKED ISSUE #55: Coding: Develop Footer
## Description
As a user,
I need the following footer mock-up to be implemented,
so that I can submit any feedback or query, follow on social media, and contact open source community.

## Mocks
Footer
(Only the section in the red box to be developed)
Text-Guidelines
(use footer text for body and small header for footer item titles)
Screenshot (159)
Screenshot (160)
Screenshot (161)
NOTE:
Background color: Rich Black (#101820 )
Button color: Caribbean (#54BCEB)
Text color: white (#FFFFFF)
The margin between text input fields and button in the feedback form is 8pt.
[Design Link for Development](https://xd.adobe.com/view/ba1323d0-760e-4f29-4658-c83848c894ba-cc22/)
### Update [Required]
- [ ] The first input field in feedback form takes email id as input and the second field is for taking queries and feedback, and below is a submit button.
- [ ] When a user clicks on the Linkedin icon, he/she should be redirected [here](https://www.linkedin.com/company/anitab-org/)

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 1 mentor.

### PR DESCRIPTION
**Pull request by tichnas**
### Description

A footer section is added which includes a form for feedback/suggestions, social icons and contact information.

Fixes #55 

### Type of Change:

- Code
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

**Screenshot**
Screenshot_2020-06-24 AnitaB org open source

### CODE REVIEW

File: src/Components/Footer/Contact.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { View, Text } from 'react-native';
+import styles from './styles';
+
+function Contact() {
+  return (
+    <View style={[styles.col, styles.right]}>
+      <Text style={[styles.text, styles.heading]}>Contact Us</Text>
+      <Text style={[styles.text, styles.description]}>
+        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nunc interdum
```
Comment (annabauza) [inline src/Components/Footer/Contact.js:10]: no `Lorem ipsum dolor` will be merge. @sakshi1499 do we have content team issue for that?

Comment (Sonal240) [reply src/Components/Footer/Contact.js:10]: I'll create it by today...

Comment (sakshi1499) [reply src/Components/Footer/Contact.js:10]: @annabauza no content team issue for footer has been created yet. Thank you @sonal240

**Comment by isabelcosta** [comment]
@tichnas can you provide a screenshot or GIF of the visual changes you've done with this PR? That would help reviewers, know what to expect to see :)

**Comment by tichnas** [comment]
Sorry about that @isabelcosta, I've updated the description.

File: src/Components/Footer/Contact.js (hunk 1 above)
Comment (annabauza) [reply src/Components/Footer/Contact.js:10]: @tichnas the content has been merged

File: src/Components/Footer/styles.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  container: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    backgroundColor: '#101820',
+    width: '100%',
```
Comment (annabauza) [inline src/Components/Footer/styles.js:8]: add `flexWrap: 'wrap',` and footer should become responsive other then that we use 80% for content.

Comment (tichnas) [reply src/Components/Footer/styles.js:8]: @annabauza The measurements that were given in the issue sum up to around 75% (of wide laptop screen). Should I go with that or change the width explicitly to 80%? Also, the empty space on the left and right should be covered in the footer color or just plain white?

Comment (sakshi1499) [reply src/Components/Footer/styles.js:8]: @tichnas Add flexWrap: 'wrap'. Yes change the width to 80% now. I think footer color is what you should go for. Please push these changes.

**Comment by tichnas** [comment]
I'm not very sure if the way I used the content is correct or not. I tried to look into other files to find the best way to use that content but it appears that `sponsors.js` create an object and export a function to get that whereas `contact_us.json` is just a simple JSON file. Please guide me.

**Comment by anna4j** [comment]
@tichnas hey! so I checked the contact_us.json and it contains some links, so I think using the content directly would be a good option since you need to include on click links to various zulip channels as well. Hope that helps!

**Pull request merged**